tests/tools.vitruv.applications.pcmumlclass.tests/META-INF/MANIFEST.MF

```diff
@@ -13,6 +13,7 @@ Require-Bundle: tools.vitruv.testutils.vsum,
  tools.vitruv.applications.util.temporary,
  org.eclipse.xtend.lib,
  edu.kit.ipd.sdq.commons.util.java,
- org.palladiosimulator.pcm.resources
+ org.palladiosimulator.pcm.resources,
+ edu.kit.ipd.sdq.activextendannotations;bundle-version="1.5.0"
```

Comment:
Is this specific version really required?

Reply:
Removed in [413c562](https://github.com/vitruv-tools/Vitruv-CaseStudies/pull/214/commits/413c562cff8076dee85da561ae3489e75177738b)